node/src/types/chainspec/network_config.rs

```diff
@@ -27,13 +28,16 @@ pub struct NetworkConfig {
 
 impl NetworkConfig {
     /// Returns a vector of chainspec validators' public key and their stake.
-    pub fn chainspec_validator_stakes(&self) -> Vec<(PublicKey, Motes)> {
+    pub fn chainspec_validator_stakes(&self) -> BTreeMap<PublicKey, U512> {
```

Comment:
What's the reason to change the return type here?

Reply:
We were only using this in places where we ended up doing this conversion.